Tasks/PublishPipelineArtifactV1/task.json

```diff
@@ -0,0 +1,47 @@
+{
+    "id": "ECDC45F6-832D-4AD9-B52B-EE49E94659BE",
+    "name": "PublishPipelineArtifact",
+    "friendlyName": "Publish pipeline artifact",
+    "description": "Publish (upload) a file or directory as a named artifact for the current run",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/publish-pipeline-artifact",
+    "helpMarkDown": "Publish (upload) a file or directory as a named artifact for the current run. Published artifacts can be downloaded in other jobs, stages, or pipelines using the \"Download pipeline artifact\" task.",
+    "category": "Utility",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "groups": [],
+    "demands": [],
+    "preview": true,
+    "minimumAgentVersion": "2.153.1",
+    "inputs": [
+        {
+            "name": "path",
+            "aliases": [
+                "targetPath"
+            ],
+            "type": "filePath",
+            "label": "File or directory path",
+            "defaultValue": "$(Pipeline.Workspace)",
+            "required": true,
+            "helpMarkDown": "The path of the file or directory to publish. Can be absolute or relative to the default working directory. Can include [variables](https://go.microsoft.com/fwlink/?LinkID=550988), but wildcards are not supported."
+        },
+        {
+            "name": "artifactName",
```

Comment:
>artifactName [](start = 21, length = 12)

Why is v2 download pipeline artifact named "artifact" but this one named "artifactName"? The two PRs should be consistent as new tasks.

Reply:
@willsmythe

Reply:
Maybe update it to "artifact" in a PPA V2 task or alias "artifact" in V1 task :)

Reply:
yah .. should rename it at some point, but not right now :) Can you at least add an alias for "artifact" so we can document "artifact" instead of "artifactName"?

Reply:
Sounds good. Will do.

Reply:
Please add a user story for the Pipeline Artifact GA feature to make sure we don't forget to switch it around.